.github/workflows/reusable-e2e.yml

```diff
@@ -132,7 +131,7 @@ jobs:
 
       - name: Install packages
         if: steps.restore-cache.outputs.cache-hit != 'true'
-        run: yarn --no-lockfile
+        uses: ./.github/actions/install-with-retries
```

Comment:
Using the reusable action i created in #2121. It doesn't use `--no-lockfile` by default.